src/app/components/table/list-table.tsx

```diff
@@ -69,18 +76,20 @@ export const makeTilaColumn = <T extends Entity = Entity>(): Column<T> => ({
 export const makeHakutapaColumn = (
   getMatchingHakutapa: (koodiUri: string) => string | undefined,
 ): Column<Haku> => ({
-  title: 'Hakutapa',
+  title: 'haku.hakutapa',
   key: 'hakutapaNimi',
   render: (haku) => <span>{getMatchingHakutapa(haku.hakutapaKoodiUri)}</span>,
 });
 
-export const makeKoulutuksenAlkamiskausiColumn = (): Column<Haku> => ({
-  title: 'Koulutuksen alkamiskausi',
+export const makeKoulutuksenAlkamiskausiColumn = (
+  t: (s: string) => string,
```

Comment:
import { TFunction } from "react-i18next";

```suggestion
  t: TFunction,
```